route-rs-graphgen/src/codegen.rs

```diff
@@ -742,6 +742,16 @@ pub fn expr_match(input: syn::Expr, arms: Vec<(syn::Pat, syn::Expr)>) -> syn::Ex
     })
 }
 
+pub fn expr_lit_int<N>(number: N) -> syn::Expr
+where
+    N: std::string::ToString,
```

Comment:
Yeah, I assume this is a workaround without const generics, which are being added to help fix this issue.